tests/web.rs

```diff
@@ -0,0 +1,161 @@
+use wasm_bindgen::*;
```

Comment:
Why do these test need to be run in `wasm_bindgen` environment? Can't we use a mock transport?

Reply:
AFAIK with mock transport we can't know if the interface works with the contract in the wild. Testing against the real contract allow us to know if it changed.

The down side is that it can't be automated.

Reply:
I tried my best but maybe I should just scrap the test suite for ENS. I can't deploy the contracts, I don't understand what testing with mock transport would do since writing the response and the request will always succeed and I don't know how to automate the browser testing.